docs/advanced.md

```diff
@@ -617,3 +617,46 @@ try {
 }
 console.info('finish')
 ```
+
+### Using Inquirer to make interactive tools
+
+Using Inquirer together with yargs is a powerful way to make your CLI tools more interactive and responsive to the user.
```

Comment:
I think Inquirer is probably the best known and a good package, but it isn't the only package that could be used. Perhaps we could be explicit somewhere that it is an example?

e.g.
```
Using Inquirer or a similar package with yargs is a powerful way to make your CLI tools more interactive and responsive to the user.
```